common/src/main/java/com/graphicsfuzz/common/util/FragmentShaderToShaderJob.java

```diff
@@ -124,9 +126,18 @@ public static ShaderJob createShaderJob(TranslationUnit tu, IRandom generator) {
           vertexShader = Optional.of(new TranslationUnit(ShaderKind.VERTEX,
               Optional.of(ShadingLanguageVersion.ESSL_310), Arrays.asList(
               new PrecisionDeclaration("precision mediump float;"),
+              new VariablesDeclaration(new QualifiedType(BasicType.VEC4,
+                  Arrays.asList(new LayoutQualifierSequence(
+                      new LocationLayoutQualifier(0)),
+                  TypeQualifier.OUT_PARAM)),
+                  new VariableDeclInfo("_GLF_pos", null, null)),
```

Comment:
Instead of using the string literal "_GLF_pos", please add a constant to com.graphicsfuzz.util.Constants, similar to:

    public static final String GLF_COLOR = "_GLF_color";

Reply:
Done